modules/data_integrity_flag/jsx/dataIntegrityFlagIndex.js

```diff
@@ -0,0 +1,155 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import {Tabs, TabPane} from 'Tabs';
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+import SetFlagForm from './setFlagForm';
+
+class DataIntegrityFlagIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      error: false,
+      isLoaded: false,
+    };
+
+    this.fetchData = this.fetchData.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   * Additionally add hiddenHeaders to global loris variable
```

Comment:
Don't know what's going on here.. the thread is marked as resolved/deleted, but the comment still seems to be in the code?

Reply:
Ah, I think i force pushed that one back on. I'll remove it again.